vendors/nxp/boards/lpc54018iotmodule/ports/secure_sockets/iot_secure_sockets.c

```diff
@@ -474,6 +474,204 @@ int32_t SOCKETS_Send( Socket_t xSocket,
 }
 /*-----------------------------------------------------------*/
 
+#if ( configPLATFORM_SOCKET_UDP_SUPPORT == 1 )
+    int32_t SOCKETS_Bind( Socket_t xSocket,
+                          const SocketsSockaddr_t * pxAddress,
+                          Socklen_t xAddressLength )
+    {
+        int32_t xStatus = SOCKETS_ERROR_NONE;
+        SSOCKETContextPtr_t pxContext = ( SSOCKETContextPtr_t ) xSocket;
+        SOCKADDR_T xTempAddress = { 0 };
+
+        if( ( SOCKETS_INVALID_SOCKET != pxContext ) && ( NULL != pxAddress ) && WIFI_IsConnected() )
+        {
+            /* The driver code expects the endianess of the address and port in host order and then swaps before
+             * sending the connection data to the firmware. */
+            xTempAddress.sin_addr.s_addr = SOCKETS_ntohl( pxAddress->ulAddress );
+            xTempAddress.sin_family = pxAddress->ucSocketDomain;
+            xTempAddress.sin_port = SOCKETS_ntohs( pxAddress->usPort );
+            xStatus = qcom_bind( ( int ) pxContext->xSocket,
+                                 ( struct sockaddr * ) &xTempAddress,
+                                 xAddressLength );
+
+            /* Keep socket state - connected */
+            if( SOCKETS_ERROR_NONE == xStatus )
+            {
+                pxContext->ulState |= nxpsecuresocketsSOCKET_CONNECTED_FLAG;
+            }
+        }
+        else
+        {
+            xStatus = SOCKETS_SOCKET_ERROR;
+        }
+
+        return xStatus;
+    }
+/*-----------------------------------------------------------*/
+
+    int32_t SOCKETS_SendTo( Socket_t xSocket,
+                            const void * pvBuffer,
+                            size_t xDataLength,
+                            uint32_t ulFlags,
+                            const SocketsSockaddr_t * pxAddress,
+                            Socklen_t xAddressLength )
+    {
+        /* The WiFi module refuses to send data if it exceeds this threshold defined in
+         * atheros_stack_offload.h */
```

Comment:
Where this comment is addressed ?  Do we check anywhere if it is increasing the threshold?

Reply:
Not sure how this comment gets here, will remove if not relevant.